modules/ROOT/pages/proc_defining-and-creating-pipelineresources.adoc

```diff
@@ -5,78 +5,60 @@
 [id="defining-and-creating-pipelineresources_{context}"]
 = Defining and creating pipeline resources
 
-`PipelineResources` are artifacts that are used as inputs to a `Task` and can be output by a `Task`.
+Pipeline resources are artifacts that are used as inputs to a task and can be output by a task.
 
 [discrete]
 === Procedure
 
-After you create `Task`s, create `PipelineResources` that contain the specifics of the Git repository and the image registry to be used in the `Pipeline` during execution as follows:
+After you create tasks, create pipeline resources that contain the specifics of the Git repository and the image registry to be used in the pipeline during execution as follows:
 
-. Create a `resources.yaml` file and with the `PipelineResource`s that contain the specifics of the Git repository and the image registry to be used in the pipeline during execution.
-
-.. Create a `PipelineResource` that defines the Git repository for the frontend application:
+. Create a `PipelineResource` that defines the Git repository for the frontend application:
```

Comment:
I have updated this section as per Siamak's comment to use `tkn resource create` instead of `oc create`.
@siamaksade @vdemeester @hrishin @ppitonak @piyush-garg @khrm @sthaha : Request you all to please review. Thanks.
Reference JIRA Link: https://issues.redhat.com/browse/RHDEVDOCS-1615